charts/cosigned/templates/webhook/deployment_webhook.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    {{- include "cosigned.labels" . | nindent 4 }}
+    control-plane: {{ template "cosigned.fullname" . }}-webhook
+  name: {{ template "cosigned.fullname" . }}-webhook
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "cosigned.selectorLabels" . | nindent 6 }}
+      control-plane: {{ template "cosigned.fullname" . }}-webhook
+  template:
+    metadata:
+      labels:
+        {{- include "cosigned.selectorLabels" . | nindent 8 }}
+        control-plane: {{ template "cosigned.fullname" . }}-webhook
+    spec:
+      nodeSelector:
+        {{- toYaml .Values.commonNodeSelector | nindent 8 }}
+      tolerations:
+        {{- toYaml .Values.commonTolerations | nindent 8 }}
+      serviceAccountName: {{ template "cosigned.fullname" . }}-webhook
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: {{ template "cosigned.name" . }}-{{ .Values.webhook.name }}
+        image: "{{ template "cosigned.image" .Values.webhook.image }}"
+        imagePullPolicy: "{{ .Values.webhook.image.pullPolicy }}"
+        env:
+        - name: SYSTEM_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: CONFIG_LOGGING_NAME
+          value: {{ template "cosigned.fullname" . }}-webhook-logging
+        - name: METRICS_DOMAIN
+          value: sigstore.dev/cosigned
+        - name: WEBHOOK_NAME
+          value: webhook
+{{- if .Values.webhook.env }}
+{{- range $key, $value := .Values.webhook.env }}
+        - name: "{{ $key }}"
+          value: "{{ $value }}"
+{{- end }}
+{{- end }}
+        args:
+        - -secret-name={{ template "cosigned.fullname" . }}-cosign-key
+        {{- range $key, $value := .Values.webhook.extraArgs }}
+        - -{{ $key }}={{ $value }}
+        {{- end }}
+        ports:
+        - containerPort: 8443
```

Comment:
Do we want to set the metrics port here ?
```
        - containerPort: 9090
          name: metrics
          protocol: TCP 
```

Reply:
will fix that

Reply:
done, thanks!